src/detail_screen/anime.css

```
@@ -0,0 +1,267 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400&family=Roboto:wght@100;300;400&display=swap');
+
+:root {
+  --grey-1: #A9A9BA;
+  --grey-2: #6F6F7A;
+  --grey-3: #35353B;
+  --grey-4: #2E2E33;
+  --grey-5: #2A2A2E;
+  --purple: #9572F8;
+}
+
+body {
+  margin: 0px;
+  min-height: 100vh;
+  background-color: var(--grey-4);
+}
+
+/* --- Header --- */
+
+header {
+  height: 64px;
+  background-color: var(--grey-2);
+  box-shadow: 0px 2px 4px 3px var(--grey-5);
+}
+
+header > div {
+  margin: auto;
+  height: 64px;
+  max-width: 1280px;
+  background-color: var(--grey-1);
+}
+
+/* --- Main --- */
+
+main {
+  margin: auto;
+  max-width: 1280px;
+  min-height: calc(100vh - 64px);
+
+  display: flex;
+  padding: 16px 0px;
+  box-sizing: border-box;
+  justify-content: space-between;
+  gap: 16px;
+}
+
+.paper {
+  padding: 16px 24px;
+  border-radius: 10px;
+  background-color: var(--grey-3);
+  box-shadow: 2px 2px 4px 3px var(--grey-5);
+}
+
+main > section:first-child {
+  width: 300px;
+}
+
+#title {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#title h1 {
+  font-family: 'Roboto';
```

Recomenda-se usar famílias de fontes genéricas em fontes customizadas:
https://stackoverflow.com/a/31378694/11111289
```suggestion
  ffont-family: 'Roboto', serif;
```